### PR TITLE
OBSDOCS-78: Update eventrouter docs

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OBSDOCS-78

Link to docs preview:
https://69647--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/log_collection_forwarding/cluster-logging-eventrouter#cluster-logging-eventrouter-deploy_cluster-logging-eventrouter

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cluster-logging-eventrouter-deploy.adoc [hunk 1]
```diff
@@ -98,21 +98,20 @@ objects:
                   cpu: ${CPU}
                   memory: ${MEMORY}
               volumeMounts:
-              - name: config-volume
-                mountPath: /etc/eventrouter
+               - name: config-volume
+                 mountPath: /etc/eventrouter
```
Comment (QiaolingTang) [inline modules/cluster-logging-eventrouter-deploy.adoc:102]:
```suggestion
              - name: config-volume
                mountPath: /etc/eventrouter
```

File: modules/cluster-logging-eventrouter-deploy.adoc [hunk 2]
```diff
@@ -98,21 +98,20 @@ objects:
...
-                name: eventrouter
+                  drop: ["ALL"]
           securityContext:
             runAsNonRoot: true
+            seccompProfile:
+              type: RuntimeDefault
+           volumes:
+             - name: config-volume
+               configMap:
+                 name: eventrouter
```
Comment (QiaolingTang) [inline modules/cluster-logging-eventrouter-deploy.adoc:114]:
```suggestion
          volumes:
          - name: config-volume
             configMap:
                name: eventrouter
```

**Comment by QiaolingTang** [comment]
LGTM.

File: modules/cluster-logging-eventrouter-deploy.adoc [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+// * logging/log_collection_forwarding/cluster-logging-eventrouter.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-eventrouter-deploy_{context}"]
 = Deploying and configuring the Event Router
 
 Use the following steps to deploy the Event Router into your cluster. You should always deploy the Event Router to the `openshift-logging` project to ensure it collects events from across the cluster.
 
-The following Template object creates the service account, cluster role, and cluster role binding required for the Event Router. The template also configures and deploys the Event Router pod. You can use this template without making changes, or change the deployment object CPU and memory requests.
+The following `Template` object creates the service account, cluster role, and cluster role binding required for the Event Router. The template also configures and deploys the Event Router pod. You can use this template without making changes, or change the deployment object CPU and memory requests.
```
Comment (Srivaralakshmi) [inline modules/cluster-logging-eventrouter-deploy.adoc:11]:
```suggestion
The following `Template` object creates the service account, cluster role, and cluster role binding required for the Event Router. The template also configures and deploys the Event Router pod. You can either use this template without making changes or edit this template to change the deployment object CPU and memory requests.
```

**Review by Srivaralakshmi — COMMENTED** [review]
@abrennan89 Good job! I have provided a few suggestions for you to consider. Please have a look at them.

Thank you!

File: modules/cluster-logging-eventrouter-deploy.adoc [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+// * logging/log_collection_forwarding/cluster-logging-eventrouter.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-eventrouter-deploy_{context}"]
 = Deploying and configuring the Event Router
 
 Use the following steps to deploy the Event Router into your cluster. You should always deploy the Event Router to the `openshift-logging` project to ensure it collects events from across the cluster.
 
-The following Template object creates the service account, cluster role, and cluster role binding required for the Event Router. The template also configures and deploys the Event Router pod. You can use this template without making changes, or change the deployment object CPU and memory requests.
+The following `Template` object creates the service account, cluster role, and cluster role binding required for the Event Router. The template also configures and deploys the Event Router pod. You can either use this template without making changes, or edit the template to change the deployment object CPU and memory requests.
```
Comment (Srivaralakshmi) [inline modules/cluster-logging-eventrouter-deploy.adoc:11]:
```suggestion
The following `Template` object creates the service account, cluster role, and cluster role binding required for the Event Router. The template also configures and deploys the Event Router pod. You can either use this template without making changes or edit the template to change the deployment object CPU and memory requests.
```
@abrennan89 Comma is not necessary here.

**Review by Srivaralakshmi — COMMENTED** [review]
@abrennan89 Thanks for the changes. Just a small nitpick. Please address it. Rest LGTM.

As I am new to the peer review squad, I would ask @bmcelvee to give my review the go-ahead. Please wait until Brandi applies the peer-review-done label.

**Review by bmcelvee — APPROVED** [review]
LGTM!

**Pull request merged**